ihatemoney/models.py

```diff
@@ -158,7 +158,9 @@ def monthly_stats(self):
         """
         monthly = defaultdict(lambda: defaultdict(float))
         for bill in self.get_bills_unordered().all():
-            monthly[bill.date.year][bill.date.month] += bill.converted_amount
+            if (bill.is_reimbursement == False):
+            # if (not bill.what.startswith("R-")):
+                monthly[bill.date.year][bill.date.month] += bill.converted_amount
```

Comment:
```suggestion
            if (not bill.is_reimbursement):
                monthly[bill.date.year][bill.date.month] += bill.converted_amount
```

Remove the comment, and don't compare boolean